src/eherkenning/backends.py

```diff
@@ -14,6 +14,14 @@ class eHerkenningBackend(_eHerkenningBackend):
     Custom backend to identify users based on the KvK number instead of RSIN
     """
 
+    def _sync_eherkenning_session_context(self, request, user):
```

Comment:
For the OIDC backend you changed it to `_persist_eherkenning_params_to_session`. Should be the same (I should have marked this in the first review as well)